Feature/gender sensitive language

JoeyStk opened the pull request:
### Short description
I changed our German translation file to be more gender sensitive.

### Proposed changes

- I changed certain words to be more gender sensitive. Please check out the discussion we had inside the issue description.
- I changed "promo team" and "municipality team" to their current names "marketing team" and "service team"
- I changed the translation of Belarus from "Weißrussland" to Belarus. For more context please checkout the discussion inside the issue and the links I've posted there.
- I only changed the German translation file and didn't have a look at the Dutch translations, since they aren't relevant yet and we probably would need an expert for this. 

### Resolved issues

Fixes: #1456

ulliholtgrave commented:
Thanks for the PR :+1: In general I like the idea about adding this to our project, but there is always some conflict between design and this conventions (e.g. text gets longer -> unwanted line-break, cognitive overload for new user etc.). Therefore, I would vouch for putting this on-hold for the moment and maybe do a test release in the next month. Then UI/UX can take a look at it and if I remember correctly they are also doing a redesign of some of our views which could also incorporate a gender-sensitive texting.

JoeyStk commented:
Yes, sounds good to me! :)

timoludwig commented on integreat_cms/__init__.py:
```diff
@@ -1 +1 @@
-__version__ = "2022.8.3"
+__version__ = "2022.8.2"
```
Could you remove commit dd806643b1f60e8f230a93ec21e7daf331559da5 from the git history of your branch? This version bump commit was probably reverted accidentally?

timoludwig commented on integreat_cms/locale/de/LC_MESSAGES/django.po:
```diff
@@ -3210,7 +3210,7 @@ msgstr "Django auth Gruppe"
 #: cms/models/users/role.py:31
 msgid "staff role"
-msgstr "Mitarbeiterrolle"
+msgstr "Teammitgliederrolle"
```
```suggestion
msgstr "Mitgliederrolle"
```

timoludwig commented on integreat_cms/locale/de/LC_MESSAGES/django.po:
```diff
@@ -3226,15 +3226,16 @@ msgstr "Rollen"
 #: cms/models/users/user.py:57
 msgid "The regions to which the user has access"
-msgstr "Die Regionen, auf die der Benutzer Zugriff hat"
+msgstr "Die Regionen, auf die der*die Benutzer*in Zugriff hat"
```
```suggestion
msgstr "Die Regionen, auf die zugegriffen werden kann"
```

timoludwig commented on integreat_cms/locale/de/LC_MESSAGES/django.po:
```diff
@@ -3226,15 +3226,16 @@ msgstr "Rollen"
 msgstr ""
-"Dies erlaubt dem Benutzer, alle Seiten zu bearbeiten und zu veröffentlichen, "
-"für die die Organisation als zuständige Organisation eingetragen ist"
+"Dies erlaubt dem*der Benutzer*in, alle Seiten zu bearbeiten und zu "
```
```suggestion
"Dies erteilt die Berechtigung, alle Seiten zu bearbeiten und zu "
```

timoludwig commented on integreat_cms/locale/de/LC_MESSAGES/django.po:
```diff
@@ -3243,12 +3244,12 @@ msgstr "Datum des letzten Chat-Besuchs"
 msgstr ""
-"Das Datum und die Uhrzeit, wann der Benutzer den Chat das letzte Mal gelesen "
-"hat"
+"Das Datum und die Uhrzeit, wann der*die Benutzer*in den Chat das letzte Mal "
+"gelesen hat"
```
```suggestion
"Das Datum und die Uhrzeit, wann der Chat das letzte Mal "
"gelesen wurde"
```

timoludwig commented on integreat_cms/locale/de/LC_MESSAGES/django.po:
```diff
@@ -3273,11 +3274,11 @@ msgstr ""
 #: cms/models/users/user.py:217
 msgid "users"
-msgstr "Benutzer"
+msgstr "Benutzer*in"
```
```suggestion
msgstr "Benutzer*innen"
```

timoludwig commented on integreat_cms/locale/de/LC_MESSAGES/django.po:
```diff
@@ -5949,12 +5951,12 @@ msgstr "Nutzerverwaltung"
 #: cms/templates/users/user_list.html:16
 msgid "Create user"
-msgstr "Benutzer erstellen"
+msgstr "Benutzer*innen erstellen"
```
```suggestion
msgstr "Benutzer*in erstellen"
```

timoludwig commented on integreat_cms/locale/de/LC_MESSAGES/django.po:
```diff
@@ -6720,47 +6723,49 @@ msgstr "Die Seite \"{page}\" wurde erfolgreich verschoben."
 #: cms/views/pages/page_actions.py:606
 msgid "Success: The user \"{}\" can now publish this page."
-msgstr "Erfolg: Der Nutzer \"{}\" kann dieses Seite nun veröffentlichen"
+msgstr "Erfolg: Der*Die Nutzer*in \"{}\" kann dieses Seite nun veröffentlichen"
```
```suggestion
msgstr "Erfolg: \"{}\" kann dieses Seite nun veröffentlichen"
```

timoludwig commented on integreat_cms/locale/de/LC_MESSAGES/django.po:
```diff
@@ -6720,47 +6723,49 @@ msgstr "Die Seite \"{page}\" wurde erfolgreich verschoben."
 "but has the implicit permission to edit this page anyway."
 msgstr ""
-"Information: Die Berechtigung des Nutzers \"{}\" um diese Seite zu "
+"Information: Die Berechtigung des*der Nutzers \"{}\" um diese Seite zu "
```
```suggestion
"Information: Die Berechtigung von \"{}\" um diese Seite zu "
```

timoludwig commented on integreat_cms/locale/de/LC_MESSAGES/django.po:
```diff
@@ -6720,47 +6723,49 @@ msgstr "Die Seite \"{page}\" wurde erfolgreich verschoben."
 "page, but has the implicit permission to publish this page anyway."
 msgstr ""
-"Information: Die Berechtigung des Nutzers \"{}\" um diese Seite zu "
+"Information: Die Berechtigung des*der Nutzer*in \"{}\" um diese Seite zu "
```
```suggestion
"Information: Die Berechtigung von \"{}\" um diese Seite zu "
```

timoludwig commented on integreat_cms/locale/de/LC_MESSAGES/django.po:
```diff
@@ -7098,44 +7103,45 @@ msgstr "Statistiken sind für diese Region nicht aktiviert."
 msgid "Superuser permissions need to be set by another superuser."
 msgstr ""
-"Administratorrechte müssen von einem anderen Administrator vergeben werden."
+"Administrator*innen-Rechte müssen von einem anderen Administrator vergeben "
```
```suggestion
"Administrations-Rechte müssen von einem anderen Administrator vergeben "
```

timoludwig reviewed (commented):
@JoeyStk Thanks, looks good! :+1:
I still found a few missing strings, search the translation file for the following:
- "Expertenmodus" -> "Expertisen-Modus"
- "Benutzer, die nur Zugriff" -> "Benutzer*innen, die nur Zugriff"
- "Benutzereinstellungen" -> "Konto-Einstellungen"
- "Terminplaner" -> "Terminplanung"
- "Nutzer" -> "Benutzer\*in" (I wouldn't mix "Nutzer\*in and "Benutzer\*in")
- "einem externen Übersetzer" -> "extern"

@ulliholtgrave Which changes are the ones where you fear UX problems? It probably makes sense to specifically ask for feedback for these, as it is almost impossible to reach all possible texts by clicking through the CMS...
The only point I see as critical is the "Benutzer'innen" menu item, but I don't see a good alternative and think it's worth the trade-off of slightly worse readability...

JoeyStk commented:
Thank you very much for your review.
I incorporated the requested changes. I also found a few more cases myself. The most challenging one was to find an alternative for "der Kontonutzer". 
Since we agreed on not wanting to use "_der*die_ Konto-Benutzer*in", I changed it to "kontoführende Person". I'm not really happy about this solution, but I couldn't come up with a better idea. Since all synonyms that came to mind had the same issue (like Inhaber*in). 
I also changed the translation of "Flag to indicate a translation is being updated by an external translator" from "_Feld_, die anzeigt, dass eine Übersetzung extern aktualisiert wird" to "_Flagge_, die anzeigt, dass eine Übersetzung extern aktualisiert wird". Just let me know, if you prefer "Flagge, _das_ anzeigt, dass eine Übersetzung extern aktualisiert wird", then I can reverse or change it.

timoludwig commented on integreat_cms/locale/de/LC_MESSAGES/django.po:
```diff
@@ -1917,23 +1917,27 @@ msgstr "Berechtigungen"
 msgid "This grants the user all permissions of the selected team"
-msgstr "Dies gewährt dem Benutzer alle Berechtigungen des ausgewählten Teams"
+msgstr ""
+"Dies gewährt der kontoführenden Person alle Berechtigungen des ausgewählten "
```
```suggestion
"Dies gewährt alle Berechtigungen des ausgewählten "
```

timoludwig commented on integreat_cms/locale/de/LC_MESSAGES/django.po:
```diff
@@ -1944,36 +1948,36 @@ msgid ""
-"Wählen Sie diese Option, um ein inaktives Nutzerkonto zu erstellen und einen "
-"Link zu dessen Aktivierung per Email an den Nutzer zu senden."
+"Wählen Sie diese Option, um ein inaktives Konto zu erstellen und einen Link "
+"zu dessen Aktivierung per Email an die entsprechende Person zu senden."
```
```suggestion
"zu dessen Aktivierung per Email zu versenden."
```

timoludwig commented on integreat_cms/locale/de/LC_MESSAGES/django.po:
```diff
@@ -2084,9 +2088,7 @@ msgstr "wird derzeit übersetzt"
-msgstr ""
-"Feld, die anzeigt, dass eine Übersetzung von einem externen Übersetzer "
-"aktualisiert wird"
+msgstr "Flagge, die anzeigt, dass eine Übersetzung extern aktualisiert wird"
```
```suggestion
msgstr "Feld, das anzeigt, dass eine Übersetzung extern aktualisiert wird"
```
_Flagge_ in Gerrman is not really a term for a boolean field, right? If you want a more literal translation, I'd rather change the english text to "Field [...]".

timoludwig reviewed (commented):
Thanks! :+1:

ulliholtgrave commented on integreat_cms/locale/de/LC_MESSAGES/django.po:
```diff
@@ -2324,7 +2326,7 @@ msgstr "als gelesen markiert von"
 #: cms/models/feedback/feedback.py:90
 msgid "The user who marked this feedback as read."
-msgstr "Der Benutzer, der dieses Feedback als gelesen markiert hat."
+msgstr "Das Konto, das dieses Feedback als gelesen markiert hat."
```
```suggestion
msgstr "Die Person, die dieses Feedback als gelesen markiert hat."
```

ulliholtgrave commented on integreat_cms/locale/de/LC_MESSAGES/django.po:
```diff
@@ -3250,37 +3252,36 @@ msgstr "Rollen"
 "and status information for broken links"
 msgstr ""
-"Aktiviert den Expertenmodus und ermöglicht zusätzliche Features wie XLIFF-"
+"Aktiviert den Expertisen-Modus und ermöglicht zusätzliche Features wie XLIFF-"
```
```suggestion
"Aktiviert den Expert*innen-Modus und ermöglicht zusätzliche Features wie XLIFF-"
```

timoludwig replied:
@ulliholtgrave I don't want to impose my opinion on you, but e.g. the [Duden](https://www.duden.de/sprachwissen/sprachratgeber/gendern-komposita-personenbezeichnungen) recommends not to gender compound words, but to find neutral alternatives instead.
And somehow I agree, I find "Expertisen-Modus" better readable than "Expert*innen-Modus"...
But if that's just my opinion, we sure can also use your suggestion.

timoludwig replied:
Or we think in a completely different direction and use e.g. "erweiterte Funktionen freischalten"?

JoeyStk replied:
Vijay, who I'm doing this PR in cooperation with, also preferred neutral words in general and "Expertise-Modus" in this specific case.

ulliholtgrave commented on integreat_cms/locale/de/LC_MESSAGES/django.po:
```diff
@@ -6009,22 +6012,22 @@ msgstr "Öffnen Sie den Seiten-Editor indem Sie auf eine Seite klicken."
 #: cms/templates/users/user_form.html:130
 msgid "Please confirm that you really want to delete this user"
-msgstr "Bitte bestätigen Sie, dass dieser Benutzer gelöscht werden soll"
+msgstr "Bitte bestätigen Sie, dass diese*r Benutzer*in gelöscht werden soll"
```
```suggestion
msgstr "Bitte bestätigen Sie, dass dieses Konto gelöscht werden soll"
```

ulliholtgrave commented on integreat_cms/locale/de/LC_MESSAGES/django.po:
```diff
@@ -6009,22 +6012,22 @@ msgstr "Öffnen Sie den Seiten-Editor indem Sie auf eine Seite klicken."
 #: cms/templates/users/user_form.html:135
 msgid "Delete this user"
-msgstr "Diesen Benutzer löschen"
+msgstr "Diese*n Benutzer*in löschen"
```
```suggestion
msgstr "Dieses Konto löschen"
```

ulliholtgrave commented on integreat_cms/locale/de/LC_MESSAGES/django.po:
```diff
@@ -6009,22 +6012,22 @@ msgstr "Öffnen Sie den Seiten-Editor indem Sie auf eine Seite klicken."
 #: cms/templates/users/user_form.html:15
 msgid "Create new user"
-msgstr "Neuen Benutzer erstellen"
+msgstr "Neue*n Benutzer*in erstellen"
```
```suggestion
msgstr "Neues Konto erstellen"
```

ulliholtgrave commented on integreat_cms/locale/de/LC_MESSAGES/django.po:
```diff
@@ -6034,44 +6037,44 @@ msgstr "Dies kann nicht rückgängig gemacht werden."
 #: cms/templates/users/user_form.html:129
 msgid "Delete user"
-msgstr "Benutzer löschen"
+msgstr "Benutzer*in löschen"
```
```suggestion
msgstr "Konto löschen"
```

ulliholtgrave commented on integreat_cms/locale/de/LC_MESSAGES/django.po:
```diff
@@ -6034,44 +6037,44 @@ msgstr "Dies kann nicht rückgängig gemacht werden."
 #, python-format
 msgid "Remove this user from region %(request.region)s"
-msgstr "Diesen Benutzer von Region %(request.region)s entfernen"
+msgstr "Diese*n Benutzer*in von Region %(request.region)s entfernen"
```
```suggestion
msgstr "Dieses Konto aus Region %(request.region)s entfernen"
```

ulliholtgrave commented on integreat_cms/locale/de/LC_MESSAGES/django.po:
```diff
@@ -6034,44 +6037,44 @@ msgstr "Dies kann nicht rückgängig gemacht werden."
 msgstr ""
-"Bitte bestätigen Sie, dass dieser Benutzer von der Region %(request.region)s "
-"entfernt werden soll."
+"Bitte bestätigen Sie, dass diese*r Benutzer*in von der Region "
```
```suggestion
"Bitte bestätigen Sie, dass dieses Konto aus der Region "
```

ulliholtgrave commented on integreat_cms/locale/de/LC_MESSAGES/django.po:
```diff
@@ -6034,44 +6037,44 @@ msgstr "Dies kann nicht rückgängig gemacht werden."
 #: cms/templates/users/region_user_form.html:120
 msgid "The user will keep access to the following regions:"
-msgstr "Der Benutzer wird den Zugang zu folgenden Regionen behalten:"
+msgstr "Die Person wird den Zugang zu folgenden Regionen behalten:"
```
```suggestion
msgstr "Das Konto wird den Zugang zu folgenden Regionen behalten:"
```

ulliholtgrave commented on integreat_cms/locale/de/LC_MESSAGES/django.po:
```diff
@@ -6034,44 +6037,44 @@ msgstr "Dies kann nicht rückgängig gemacht werden."
 #: cms/templates/users/region_user_form.html:124
 msgid "Remove user from region"
-msgstr "Benutzer von Region entfernen"
+msgstr "Benutzer*in von Region entfernen"
```
```suggestion
msgstr "Konto aus Region entfernen"
```

ulliholtgrave commented on integreat_cms/locale/de/LC_MESSAGES/django.po:
```diff
@@ -6034,44 +6037,44 @@ msgstr "Dies kann nicht rückgängig gemacht werden."
 #: cms/templates/users/user_list.html:16
 msgid "Create user"
-msgstr "Benutzer erstellen"
+msgstr "Benutzer*in erstellen"
```
```suggestion
msgstr "Konto erstellen"
```

ulliholtgrave commented on integreat_cms/locale/de/LC_MESSAGES/django.po:
```diff
@@ -6096,12 +6099,12 @@ msgstr "Letzter Login"
 #: cms/templates/users/user_list.html:51
 msgid "No users found with these filters."
-msgstr "Keine Benutzer mit diesen Filtern gefunden."
+msgstr "Keine Benutzer*innen mit diesen Filtern gefunden."
```
```suggestion
msgstr "Keine Konten mit diesen Filtern gefunden."
```

ulliholtgrave commented on integreat_cms/locale/de/LC_MESSAGES/django.po:
```diff
@@ -6096,12 +6099,12 @@ msgstr "Letzter Login"
 #: cms/templates/users/user_list.html:53
 msgid "No users available yet."
-msgstr "Noch keine Benutzer vorhanden."
+msgstr "Noch keine Benutzer*innen vorhanden."
```
```suggestion
msgstr "Noch keine Konten vorhanden."
```

ulliholtgrave commented on integreat_cms/locale/de/LC_MESSAGES/django.po:
```diff
@@ -6120,31 +6123,31 @@ msgstr "Aktivierungslink erneut versenden"
 #: cms/templates/users/user_form.html:94
 msgid "Designates whether the user has access to the network management."
-msgstr "Legt fest, ob der Benutzer Zugriff auf die Netzwerkverwaltung hat."
+msgstr "Legt fest, ob die Person Zugriff auf die Netzwerkverwaltung hat."
```
```suggestion
msgstr "Legt fest, ob das Konto Zugriff auf die Netzwerkverwaltung hat."
```

ulliholtgrave commented on integreat_cms/locale/de/LC_MESSAGES/django.po:
```diff
@@ -7267,44 +7269,45 @@ msgstr "Statistiken sind für diese Region nicht aktiviert."
 #: cms/views/users/region_user_actions.py:42
 msgid "User {} was successfully deleted."
-msgstr "Benutzer {} wurde erfolgreich gelöscht."
+msgstr "Benutzer*in {} wurde erfolgreich gelöscht."
```
```suggestion
msgstr "Das Konto von {} wurde erfolgreich gelöscht."
```

ulliholtgrave commented on integreat_cms/locale/de/LC_MESSAGES/django.po:
```diff
@@ -7267,44 +7269,45 @@ msgstr "Statistiken sind für diese Region nicht aktiviert."
 #: cms/views/users/region_user_actions.py:54
 msgid "User {} was successfully removed from this region."
-msgstr "Benutzer {} wurde erfolgreich von dieser Region entfernt."
+msgstr "Benutzer*in {} wurde erfolgreich von dieser Region entfernt."
```
```suggestion
msgstr "Das Konto von {} wurde erfolgreich von dieser Region entfernt."
```

ulliholtgrave commented on integreat_cms/locale/de/LC_MESSAGES/django.po:
```diff
@@ -7267,44 +7269,45 @@ msgstr "Statistiken sind für diese Region nicht aktiviert."
 #: cms/views/users/user_form_view.py:119
 msgid "User \"{}\" was successfully created"
-msgstr "Benutzer \"{}\" wurde erfolgreich erstellt"
+msgstr "Benutzer*in \"{}\" wurde erfolgreich erstellt"
```
```suggestion
msgstr "Das Konto für \"{}\" wurde erfolgreich erstellt"
```

ulliholtgrave commented on integreat_cms/locale/de/LC_MESSAGES/django.po:
```diff
@@ -7267,44 +7269,45 @@ msgstr "Statistiken sind für diese Region nicht aktiviert."
 #: cms/views/users/user_form_view.py:127
 msgid "User \"{}\" was successfully saved"
-msgstr "Benutzer \"{}\" wurde erfolgreich gespeichert"
+msgstr "Benutzer*in \"{}\" wurde erfolgreich gespeichert"
```
```suggestion
msgstr "Das Konto von \"{}\" wurde erfolgreich gespeichert"
```

ulliholtgrave commented on integreat_cms/locale/de/LC_MESSAGES/django.po:
```diff
@@ -7267,44 +7269,45 @@ msgstr "Statistiken sind für diese Region nicht aktiviert."
 #: cms/views/users/user_actions.py:37
 msgid "User was successfully deleted"
-msgstr "Benutzer wurde erfolgreich gelöscht"
+msgstr "Benutzer*in wurde erfolgreich gelöscht"
```
```suggestion
msgstr "Konto wurde erfolgreich gelöscht"
```

ulliholtgrave commented on integreat_cms/locale/de/LC_MESSAGES/django.po:
```diff
@@ -7267,44 +7269,45 @@ msgstr "Statistiken sind für diese Region nicht aktiviert."
 msgstr ""
-"Ein Benutzer muss entweder Mitarbeiter/Administrator sein, oder mindestens "
-"einer Region zugewiesen sein."
+"Ein*e Benutzer*in muss entweder Teammitglied/Administrator*in sein, oder "
```
```suggestion
"Das Konto muss entweder Teammitglied/Administrator*in sein, oder "
```

ulliholtgrave commented on integreat_cms/locale/de/LC_MESSAGES/django.po:
```diff
@@ -7562,20 +7565,20 @@ msgstr ""
 
 #~ msgid "Activate your {} account"
-#~ msgstr "{}-Benutzerkonto aktivieren"
+#~ msgstr "{}-Account aktivieren"
```
```suggestion
#~ msgstr "{}-Konto aktivieren"
```

ulliholtgrave commented on integreat_cms/locale/de/LC_MESSAGES/django.po:
```diff
@@ -7879,10 +7883,10 @@ msgstr ""
 
 #~ msgid "User was successfully saved"
-#~ msgstr "Benutzer wurde erfolgreich gespeichert"
+#~ msgstr "Benutzer*in wurde erfolgreich gespeichert"
```
```suggestion
#~ msgstr "Konto wurde erfolgreich gespeichert"
```

ulliholtgrave commented on integreat_cms/locale/de/LC_MESSAGES/django.po:
```diff
@@ -7904,19 +7908,19 @@ msgstr ""
 
 #~ msgid "enable expert mode"
-#~ msgstr "Expertenmodus aktivieren"
+#~ msgstr "Expertise-Modus aktivieren"
```
```suggestion
#~ msgstr "Expert*innen-Modus aktivieren"
```

ulliholtgrave commented on integreat_cms/locale/de/LC_MESSAGES/django.po:
```diff
@@ -7904,19 +7908,19 @@ msgstr ""
 
 #~ msgid "expert mode"
-#~ msgstr "Expertenmodus"
+#~ msgstr "Expertise-Modus"
```
```suggestion
#~ msgstr "Expert*innen-Modus"
```

ulliholtgrave commented on integreat_cms/locale/de/LC_MESSAGES/django.po:
```diff
@@ -8225,30 +8229,31 @@ msgstr ""
 
 #~ msgid "Create User"
-#~ msgstr "Benutzer erstellen"
+#~ msgstr "Benutzer*in erstellen"
```
```suggestion
#~ msgstr "Konto erstellen"
```

ulliholtgrave commented on integreat_cms/locale/de/LC_MESSAGES/django.po:
```diff
@@ -8225,30 +8229,31 @@ msgstr ""
 
 #~ msgid "Edit User \"%(user_name)s\""
-#~ msgstr "Benutzer \"%(user_name)s\" bearbeiten"
+#~ msgstr "Benutzer*in \"%(user_name)s\" bearbeiten"
```
```suggestion
#~ msgstr "Das Konto von \"%(user_name)s\" bearbeiten"
```

ulliholtgrave commented on integreat_cms/locale/de/LC_MESSAGES/django.po:
```diff
@@ -8225,30 +8229,31 @@ msgstr ""
 
 #~ msgid "Create new User"
-#~ msgstr "Neuen Benutzer erstellen"
+#~ msgstr "Neue*n Benutzer*in erstellen"
```
```suggestion
#~ msgstr "Neues Konto erstellen"
```

ulliholtgrave commented on integreat_cms/locale/de/LC_MESSAGES/django.po:
```diff
@@ -8225,30 +8229,31 @@ msgstr ""
 
 #~ msgid "All Users"
-#~ msgstr "Alle Benutzer"
+#~ msgstr "Alle Benutzer*innen"
```
```suggestion
#~ msgstr "Alle Konten
```

ulliholtgrave commented on integreat_cms/locale/de/LC_MESSAGES/django.po:
```diff
@@ -8520,28 +8525,28 @@ msgstr ""
 
 #~ msgid "Expert mode"
-#~ msgstr "Expertenmodus"
+#~ msgstr "Expertise-Modus"
```
```suggestion
#~ msgstr "Expert*innen-Modus"
```

ulliholtgrave commented on integreat_cms/locale/de/LC_MESSAGES/django.po:
```diff
@@ -8983,10 +8988,10 @@ msgstr ""
 
 #~ msgid "User saved successfully."
-#~ msgstr "Benutzer wurde erfolgreich gespeichert."
+#~ msgstr "Benutzer*in wurde erfolgreich gespeichert."
```
```suggestion
#~ msgstr "Konto wurde erfolgreich gespeichert."
```

JoeyStk commented:
@ulliholtgrave Since Duden and the definition inside the issue prefer "Expertise-Modus" over "Expert*innen-Modus", I rejected your suggestion. Still I'm very thankful to you for reviewing my PR and making suggestions :)

timoludwig commented on integreat_cms/locale/de/LC_MESSAGES/django.po:
```diff
@@ -1960,36 +1960,36 @@ msgid ""
 msgstr ""
-"Dieser Link erlaubt es dem Nutzer ein Passwort auszuwählen und aktiviert das "
-"Konto nach Bestätigung."
+"Dieser Link erlaubt es Nutzer*innen ein Passwort auszuwählen und aktiviert "
```
```suggestion
"Dieser Link erlaubt es Benutzer*innen ein Passwort auszuwählen und aktiviert "
```
I think we could replace `Nutzer*in` with `Benutzer*in` to be more consistent in our wording (or use "Konto" if we decide on that term for a user account).

timoludwig commented on integreat_cms/locale/de/LC_MESSAGES/django.po:
```diff
@@ -8233,30 +8233,31 @@ msgstr ""
 #~ msgid "You cannot send activation links to active users"
 #~ msgstr ""
-#~ "Sie können keinen Aktivierungslink an bereits aktive Nutzer versenden."
+#~ "Sie können keinen Aktivierungslink an bereits aktive Benutzer*in "
```
```suggestion
#~ "Sie können keinen Aktivierungslink an bereits aktive Benutzer*innen "
```

timoludwig reviewed (changes requested):
Thanks a lot! :+1:
I still found two small issues:

1. There are now a few inconsistencies between "Benutzer\*in" and "Konto". I think both variants are fine, but we should choose one and use the same wording for all occurrences. (e.g. "All Users" is translated with "Alle Konten", but "All users" with "Alle Benutzer\*innen")...
2. A few strings are still not gendered:
  https://github.com/digitalfabrik/integreat-cms/blob/184f0302cd573e4c2ff19f9b343ea9013e84fc71/integreat_cms/locale/de/LC_MESSAGES/django.po#L8492
  https://github.com/digitalfabrik/integreat-cms/blob/184f0302cd573e4c2ff19f9b343ea9013e84fc71/integreat_cms/locale/de/LC_MESSAGES/django.po#L3303
  https://github.com/digitalfabrik/integreat-cms/blob/184f0302cd573e4c2ff19f9b343ea9013e84fc71/integreat_cms/locale/de/LC_MESSAGES/django.po#L2327

JoeyStk commented:
Thank you for your feedback! I will discuss this issue again with the service-team on monday. There are instances where "Konten" is strange and there are other where we would need to use "der*die Benutzer*innen", which we ruled out so far. So I think we'll go for "Konten" in all occurance, even if they might seem strange here and there. I will check with the service-team and then make the changes

timoludwig commented on integreat_cms/locale/de/LC_MESSAGES/django.po:
```diff
@@ -3258,37 +3254,36 @@ msgstr "Rollen"
 "and status information for broken links"
 msgstr ""
-"Aktiviert den Expertenmodus und ermöglicht zusätzliche Features wie XLIFF-"
+"Aktiviert den Expertisen-Modus und ermöglicht zusätzliche Features wie XLIFF-"
```
Below, it is called "Expertise-Modus" and here it is "Expertisen-Modus"... I don't really have a preference, but probably we should use the same wording in all occasions.

JoeyStk replied:
(Rejected) We landed on Expert:innenmodus

timoludwig commented on integreat_cms/locale/de/LC_MESSAGES/django.po:
```diff
@@ -2546,7 +2542,7 @@ msgstr "sichtbar"
 #: cms/models/languages/language_tree_node.py:29
 msgid "Defines whether the language is displayed to the users of the app"
-msgstr "Definiert, ob die Sprache den Nutzer:innen der App angezeigt wird"
+msgstr "Definiert, ob die Sprache den Konten der App angezeigt wird"
```
```suggestion
msgstr "Definiert, ob die Sprache den Benutzer*innen der App angezeigt wird"
```
In this case "Konten" does not make sense, since the users of our app do not have logins etc...

timoludwig commented on integreat_cms/locale/de/LC_MESSAGES/django.po:
```diff
@@ -2766,38 +2762,38 @@ msgstr ""
 #: cms/models/pages/page.py:175
 msgid "publishers"
-msgstr "Veröffentlicher"
+msgstr "Veröffentlicht von: "
```
```suggestion
msgstr "Redakteur*innen"
```
The problem is that `publishers` does not contain all people which actually published the page, but people who have the permission to publish the page (in other words - the editors), so the `editors` fields is a bit misleading at the moment, because it contains all `authors`... not your fault, I will open another issue for it.

timoludwig commented on integreat_cms/locale/de/LC_MESSAGES/django.po:
```diff
@@ -3210,7 +3206,7 @@ msgstr "tünews aktivieren"
 #: cms/models/regions/region.py:290
 msgid "Enable to show a feed of tunews articles to users."
-msgstr "Auswählen, um Benutzern einen tünews-Feed anzuzeigen."
+msgstr "Auswählen, um anderen Konten einen tünews-Feed anzuzeigen."
```
```suggestion
msgstr "Auswählen, um Benutzer*innen einen tünews-Feed anzuzeigen."
```
Here as well - I think app users are meant.

JoeyStk replied:
If I'm not mistaken this should be resolved now

timoludwig commented on integreat_cms/locale/de/LC_MESSAGES/django.po:
```diff
@@ -3090,7 +3086,7 @@ msgstr "Bei Landkreisen geben Sie die Postleitzahl des Verwaltungssitzes an."
 #: cms/models/regions/region.py:187
 msgid "email address of the administrator"
-msgstr "E-Mail-Adresse des Administrators"
+msgstr "E-Mail-Adresse des*r Administrator*in"
```
```suggestion
msgstr "E-Mail-Adresse der Verwaltung"
```
I think `des*r` is invalid German - it would have to be `des*der` which we decided to avoid...

JoeyStk replied:
(Rejected) After our talk with @osmers we landed on "%(BRANDING)-Teammitglied"

timoludwig reviewed (commented):
Thanks a lot! :+1:
I only found a few small issues...

JoeyStk commented:
Hey hey, 
so after gathering some more informations on this topic, this PR is ready for review again. 
1. Following the Integreat-guide I used ":" instead of "*"
2. "Expertise-Modus" was changed to "Expert:innenmodus", because @osmers thought "Expertise-Modus" was inaccurate.
3. The German translation "Konto" is only used for "account". The German translation "Benutzer:in" is used for "user"/"users".
4. The author chat is called "Integreat chat"
5. If necessary I changed the English original to make it more accurate.
6. I changed the role "Management" to "Manager"
7. Use "des:der" instead of "des:r"

I'm quite happy with this PR now, I think it's as consistent as possible now. There were a few instances, where I couldn't change the English translation, I think that's because they aren't really used. So I skipped them. I hope that's fine and that is not going to be the source for future issues. Please find my notes from my conversation with @timoludwig and @osmers [here](https://nextcloud.tuerantuer.org/index.php/f/2932405). I hope I more or less fulfilled the requirements. 

I'm looking forward to your feedback :)

ulliholtgrave reviewed (approved):
Thanks, I a kinda okay with the wording right now 👍

timoludwig reviewed (changes requested):
Thanks a lot! :+1:
We're getting there, but I still noticed a few problems...

timoludwig commented on integreat_cms/locale/de/LC_MESSAGES/django.po:
```diff
@@ -2516,8 +2513,8 @@ msgid ""
 msgstr ""
-"Wird Benutzern angezeigt, wenn der gespiegelte Teil einer Seite nicht in "
-"ihrer Sprache verfügbar ist."
+"Wird Konten angezeigt, wenn der gespiegelte Teil einer Seite nicht in ihrer "
```
```suggestion
"Wird Benutzer:innen angezeigt, wenn der gespiegelte Teil einer Seite nicht in ihrer "
```
Not sure why I missed this earlier, this is again on of the cases where users of the apps are addressed who do not have an account.

JoeyStk replied:
No worries. I understand :)

timoludwig reviewed (changes requested):
Thanks! Now I found only one small remaining issue! :+1:

timoludwig reviewed (approved):
Thank you! :pray:
I think that's it! :+1:
